low-level/sensors/impl/generic/node_data.py

```diff
@@ -281,8 +281,9 @@ def _get_bmc_info(self):
         try:
             bmcdata = {'ifId': 'ebmc0', 'ipV4Prev': "", 'ipV4': "", 'nwStatus': "DOWN", 'nwCableConnStatus': 'UNKNOWN'}
             ipdata = sp.Popen("sudo ipmitool lan print", shell=True, stdout=sp.PIPE, stderr=sp.PIPE).communicate()[0].decode().strip()
-            bmcip = re.findall("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", ipdata)[0]
+            bmcip = re.findall("\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}", ipdata)

```

Comment:
why changed? existing one suffice and proper too

Reply:
"List index out of range" error is coming periodically in sspl log in VM environment. after making this small change error is stoped.

Reply:
Ok